Classes/Core/Testbase.php

```diff
@@ -473,10 +514,11 @@ public function testDatabaseNameIsNotTooLong($originalDatabaseName, array $confi
      * @param array $overruleConfiguration Overrule factory and base configuration
      * @throws Exception
      */
-    public function setUpLocalConfiguration($instancePath, array $configuration, array $overruleConfiguration): void
+    public function setUpLocalConfiguration(string $instancePath, array $configuration, array $overruleConfiguration): void
     {
         // Base of final LocalConfiguration is core factory configuration
-        $finalConfigurationArray = require ORIGINAL_ROOT . 'typo3/sysext/core/Configuration/FactoryConfiguration.php';
+        $coreExtensionPath = $this->composerPackageManager->getPackageInfo('typo3/cms-core')->getRealPath();
+        $finalConfigurationArray = require $coreExtensionPath . '/Configuration/FactoryConfiguration.php';
         $finalConfigurationArray = array_replace_recursive($finalConfigurationArray, $configuration);
         $finalConfigurationArray = array_replace_recursive($finalConfigurationArray, $overruleConfiguration);
         $result = file_put_contents(
```

Comment:
```suggestion
        $result = @file_put_contents(
```

Reply:
Not sure if we should really suppress the file_put_contents() here. Anyway, could be a dedicated change. A change should only contain needed and related things and not starting to do a lot of unrelated side changes. But will have a detailed look later.

Reply:
Have added the silence-operator.